src/components/organisms/PublicFooter.vue

```diff
@@ -39,7 +39,7 @@ const actualYear = computed(() => new Date().getFullYear());
         </div>
 
         <div class="column">
-          <DcLocalePicker />
+          <DcLocalePicker style="width: 170px" />
```

Comment:
I've seen more than once inline styles now. Unfortunetally this is not awalys a good decision since sometimes by a "design decision" the size may suffer changes and this is never too clear why we set those "magic numbers". If possible might be a good strategy to have classes.

Reply:
I agree, so I've change this into a class for now